src/Lecture2.hs

```diff
@@ -178,7 +192,10 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing []       = True
+isIncreasing [_]      = True
+isIncreasing [x, y]   = x < y
+isIncreasing (x : xs) = x < head xs && isIncreasing xs
```

Comment:
You can avoid using unsafe functions by using nested pattern matching. And after that, you can remove a redundant case.

```suggestion
isIncreasing (x : y : ys) = x < y && isIncreasing (y : ys)
```